src/components/mols/CreditCardPayment/index.tsx

```diff
@@ -69,17 +70,30 @@ const CreditCardPayment: React.FC<ICreditCardPaymentProps> = ({
       ),
   });
   const initialValues: ICreditCardPayment = {
-    cardNumber: "",
+    creditCard: "",
     cvv: "",
-    nameOnCard: "",
+    fullName: "",
     cardMonth: "1",
     cardYear: String(new Date().getFullYear()),
     TermsOfUse: true,
   };
 
   const handleSubmit = async (values: ICreditCardPayment) => {
+    const fragmentedName = values.fullName.split(" ");
+    const firstName = fragmentedName[0];
+    const lastName = fragmentedName.slice(1).join(" ");
```

Comment:
Pedro Henrique Beserra Lima

My Last name is _Henrique Beserra Lima_ or _Henrique_? 

Change this to:

```suggestion
    const lastName = fragmentedName.at(-1)  //or fragmentedName[fragmentedName.length - 1];
```

Reply:
Done.